docs/cennznet/nft.md

```diff
@@ -305,3 +307,203 @@ The following sections contain the module details.
 - **interface**: `api.rpc.nft.collectedTokens`
 - **jsonrpc**: `nft_collectedTokens`
 - **summary**: Get the tokens owned by an address in a certain collection
+ 
+# Derive queries
+
+- **interface**: api.derive.nft.function_name
+# Module: nft/collectionInfo
+
+## Table of contents
+
+## Functions
+
+### collectionInfo
+
+▸ **collectionInfo**(`instanceId`, `api`): () => `Observable`<[`CollectionInfo`](../interfaces/nft_types.collectioninfo.md)[]\>
+
+Get map of collection id to collection name
+
+#### Parameters
+
+| Name | Type |
+| :------ | :------ |
+| `instanceId` | `string` |
+| `api` | `ApiInterfaceRx` |
+
+#### Returns
+
+`fn`
+
+[CollectionInfo](../interfaces/nft_types.collectioninfo.md)
+
+▸ (): `Observable`<[`CollectionInfo`](../interfaces/nft_types.collectioninfo.md)[]\>
+
+##### Returns
+
+`Observable`<[`CollectionInfo`](../interfaces/nft_types.collectioninfo.md)[]\>
+
+#### Defined in
+
+[packages/api/src/derives/nft/collectionInfo.ts:13](https://github.com/cennznet/api.js/blob/7367fb0/packages/api/src/derives/nft/collectionInfo.ts#L13)
+
+# Module: nft/openCollectionListings
+
+## Table of contents
+
+## Functions
+
+### openCollectionListings
+
+▸ **openCollectionListings**(`instanceId`, `api`): (`collectionId`: `string`) => `Observable`<[`DeriveTokenInfo`](../interfaces/nft_types.derivetokeninfo.md)[]\>
+
+Gets all tokens in a collection that have an open listing
+
+#### Parameters
+
+| Name | Type |
+| :------ | :------ |
+| `instanceId` | `string` |
+| `api` | `ApiInterfaceRx` |
+
+#### Returns
+
+`fn`
+
+List of tokens that have open listings in given collection
+
+▸ (`collectionId`): `Observable`<[`DeriveTokenInfo`](../interfaces/nft_types.derivetokeninfo.md)[]\>
+
+##### Parameters
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `collectionId` | `string` | The collection Id value |
+
+##### Returns
+
+`Observable`<[`DeriveTokenInfo`](../interfaces/nft_types.derivetokeninfo.md)[]\>
+
+#### Defined in
+
+[packages/api/src/derives/nft/openCollectionListings.ts:30](https://github.com/cennznet/api.js/blob/7367fb0/packages/api/src/derives/nft/openCollectionListings.ts#L30)
+
+# Module: nft/tokenInfo
+
+## Table of contents
+
+## Functions
+
+### tokenInfo
+
+▸ **tokenInfo**(`instanceId`, `api`): (`tokenId`: `TokenId` \| `EnhancedTokenId`) => `Observable`<[`DeriveTokenInfo`](../interfaces/nft_types.derivetokeninfo.md)\>
```

Comment:
I think we should avoid emitting this `instanceId`, `api` part since it's decorated away in the real thing and doesn't help user at all
```suggestion
▸ **tokenInfo**(`tokenId`: `TokenId` \| `EnhancedTokenId`) => `Observable`<[`DeriveTokenInfo`](../interfaces/nft_types.derivetokeninfo.md)\>
```

Reply:
ya... had similar thought... will see